index.html

```diff
@@ -222,8 +235,63 @@ <h3>tccc mode</h3>
     <p>Overlapping more than two notes is undefined behavior in tccc mode. Don't do it.</p>
   </div>
 
+  <div id="pitchbend" class="container">
+    <h2>Pitch Bend Events</h2>
+
+    <p>TCCC can perform additional adjustments to the pitch of notes using MIDI Pitch Bend events, allowing you to 
+      set notes to microtonal values.</p>
+
+    <ul>
+      <li>In your DAW, MIDI pitch bend events are a numerical value that is converted to a change in pitch 
+        by the output device (e.g. your synthesizer plugin, soundfont player, etc).</li>
+      <ul>
+        <li>Tutorials if you are unfamiliar with MIDI pitch bends:</li>
+        <ul>
+          <li>Reaper: 
+            <a href="https://www.youtube.com/watch?v=4rOsXnQSoT8">Federico Bonacossa</a></li>
+          <li>Ableton: 
+            <a href="https://quadrophone.com/synthesis/using-pitch-bend/">Using Pitch Bend - Quadrophone</a></li>
+          <li>FL Studio: 
+            <a href="https://macprovideo.com/article/audio-software/fl-studio-automation-clips-vs-channel-editing-vs-per-note-editing">
+            Macprovideo (use the Channel Editing method)</a></li>
+        </ul>
```

Comment:
I'm not a huge fan of adding external resources here, since these links aren't guaranteed to work forever, and just generally I don't want to promote specific resources. Plus, I believe people should figure out their DAW on their own. It helps the documentation remain reliable in the long term if we don't add anything that could be subject to changes in preference, i.e. if another DAW becomes popular, we'd have to update this.

Reply:
Updated tutorial to remove these links: 1c6745acfe353ba194f60a47e548da83353aeedf